HACKING.md

```diff
@@ -32,9 +32,24 @@ You can install the daily build PPA by running:
 We'd love the help!
 
 - Submit pull requests against [snapcraft](https://github.com/ubuntu-core/snapcraft/pulls)
+- Make sure to read the [contribution guide](CONTRIBUTING.md)
 - Our mailing list is snappy-devel@lists.ubuntu.com
 - We can also be found on the #snappy IRC channel on Freenode
 
+
+### Staging server
+
+Snapcraft has the ability to upload .snaps for publication in the Snappy Store.
+If you're working on a feature that involves that stuff, you might want to use
+the staging server instead of the production store. To do that, make sure you
+have an account on the [staging server](https://login.staging.ubuntu.com), then
+make sure you have the following environment variables defined:
+
+    UBUNTU_STORE_API_ROOT_URL='https://myapps.developer.staging.ubuntu.com/dev/api/'
```

Comment:
Not a bad idea. I'm not sure how often this will be used, honestly-- it's mostly here so I don't need to write it down somewhere else :stuck_out_tongue: .